.gitlab-ci-prod.yml

```diff
@@ -0,0 +1,49 @@
+image: python:3.6
+
+stages:
+  - terraform
+  - deploy
+  - functional_tests
+
+before_script:
+  - apt-get -y update
+  - apt-get -y install jq moreutils gettext unzip zip
+  - pip install -r requirements-dev.txt
+  - mkdir ./external_binaries
+  - curl https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_linux_amd64.zip -o /tmp/terraform.zip
+  - unzip /tmp/terraform.zip -d ./external_binaries/
+  - rm /tmp/terraform.zip
+  - export PATH=$PATH:`pwd`/external_binaries/
+  - export DEPLOYMENT_STAGE=$CI_COMMIT_REF_NAME
+  - if [ "$DEPLOYMENT_STAGE" == "master" ]; then
```

Comment:
Why do we need references to the other environments in gitlab file for prod?

Reply:
Removing